capture_resource_metrics.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+#
+# PREREQUISITES
+# - jq
+# - oc (logged in at the cmd line in order to get the bearer token)
+# - a "perf-test-start-time.txt" file with a valid rfc3339 timestamp from a moment before performance tests started 
+#   (it can be produced with this multiplatform command: `date -u +%Y-%m-%dT%TZ > perf-test-start-time.txt`)
+# - a "perf-test-end-time.txt" file with a valid rfc3339 timestamp from a moment after performance tests finished
+#
+# VARIABLES
+START_TIME_FILENAME=${START_TIME_FILENAME:-perf-test-start-time.txt}
+END_TIME_FILENAME=${END_TIME_FILENAME:-perf-test-end-time.txt}
```

Comment:
It might be worth adding a .gitignore to this repo for `perf-test-start-time.txt`, `perf-test-start-time.txt`, and `.idea/`.